src/app/components/formular-gui/formular-gui.component.html

```diff
@@ -0,0 +1,122 @@
+<div class="container">
+  <div *ngFor="let formModel of formModels" class="control-margin" [ngSwitch]="formModel.constructor.name">
+    <div class="row" *ngSwitchCase="'Button'">
+      <button class="btn btn-primary" [id]="formModel.id" [value]="formModel.id" [ngModel]="formModel.value" [type]="formModel.type"
+        (ngModelChange)="onModelChange($event, formModel.id)">{{ formModel.value }}</button>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Textfield'">
+      <label for="formModel.id">{{ formModel.value }}</label>
+      <input [id]="formModel.id" [type]="formModel.type" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.id)">
+    </div>
+
+    <div class="row" *ngSwitchCase="'Label'">
+      <label [id]="formModel.id">{{ formModel.value }}</label>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Checkbox'">
+      <label for="formModel.id">{{ formModel.value }}</label>
+      <input type="checkbox" [id]="formModel.id" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.id)">
+    </div>
+
+    <div class="row" *ngSwitchCase="'Textarea'">
+      <label for="formModel.id">{{ formModel.value }}</label>
+      <textarea [id]="formModel.id" [rows]="formModel.lines" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.id)"></textarea>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Combobox'">
+      <label for="formModel.id">{{ formModel.value }}</label>
+      <select [id]="formModel.id" class="chosen-select" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.value)">
+        <option *ngFor="let option of formModel.options" [id]="option.id" [value]="option.value">{{ option.value }}</option>
+      </select>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Separator'">
+      <hr>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Hbox'">
+      <ul class="flex-container hbox">
+        <div *ngFor="let control of formModel.controls" [ngSwitch]="control.constructor.name">
+          <li class="flex-item" *ngSwitchCase="'Button'">
```

Comment:
Alle Controls wiederholen sich -> duplicate Code. Es gibt zwei Möglichkeiten das zu lösen:
* Den Inhalt der For-Schleife in eine eigene Komponente auslagern
* Ein Template für den Inhalt der For-Schleife anlegen, und dieses entsprechend einbinden